### DISCUSSION TITLE
Pages support

### DISCUSSION THREAD

**Discussion by mariusa** [vendurehq/vendure#788] [General]
Hi, is support for Online Store Pages planned? (pages like About us, Contact...) With i18n support.

Or, what's the recommended way to set this up? Integration with Strapi?

**Comment by michaelbromley** [comment]
Hi, this is not something we're planning. The reason is that there are already so many good headless CMS options out there - Strapi, Sanity, StoryBlok, Contentful etc. Doing CMS right is not easy (i18n, drafts, versioning etc), so delegating this task to the specialists allows us to concentrate on the core e-commerce aspects of Vendure.

**Reply by mariusa** [reply]
That makes sense, thanks for clarification
